docs/source/azure/examples/removepublicstorageips.rst

```diff
@@ -0,0 +1,22 @@
+Remove public storage IP-s
+==========================
+
+.. code-block:: yaml
+
+    policies:
+        - name: storage-no-public-ip
+        description: |
+            find publicly available storage ip-s and remove them keeping virtual network access intact
```

Comment:
Is `ip-s` a typo?

(minor) "...and remove them (while) keeping virtual network access intact"

Reply:
how do you say ip in plural form?